src/node_crypto.cc

```diff
@@ -2998,7 +2998,7 @@ CipherBase::UpdateResult CipherBase::Update(const char* data,
   int buff_len = len + EVP_CIPHER_CTX_block_size(ctx_.get());
   // For key wrapping algorithms, get output size by calling
   // EVP_CipherUpdate() with null output.
-  if (mode == EVP_CIPH_WRAP_MODE &&
+  if (kind_ == kCipher && mode == EVP_CIPH_WRAP_MODE &&
```

Comment:
Is this change intended? If so, this likely should be a different commit.

Reply:
@jasnell this is needed. Sorry the title of this commit is a little misleading but I explained that I need the code change in the description